plugins/woocommerce/src/Internal/Utilities/HtmlSanitizer.php

```diff
@@ -32,6 +32,20 @@ class HtmlSanitizer {
 		),
 	);
 
+	/**
+	 * Sanitizes a chunk of HTML, by following the same rules as `wp_kses_post()` but also allowing
+	 * the style element to be supplied.
+	 *
+	 * @param string $html
```

Comment:
```suggestion
	 * @param string $html The HTML to be sanitized.
```

Satisfy linter/describe the purpose of this parameter.